lib/modules/manager/argocd/extract.ts

```diff
@@ -66,28 +115,13 @@ function createDependency(
   };
 }
 
-export function extractPackageFile(
-  content: string,
-  fileName: string,
-  _config?: ExtractConfig
-): PackageFileContent | null {
-  // check for argo reference. API version for the kind attribute is used
-  if (fileTestRegex.test(content) === false) {
-    return null;
-  }
-
-  let definitions: ApplicationDefinition[];
-  try {
-    definitions = loadAll(content) as ApplicationDefinition[];
-  } catch (err) {
-    logger.debug({ err, fileName }, 'Failed to parse ArgoCD definition.');
-    return null;
+export function processMultipleSources(
+  sources: Array<ApplicationSource>
+): Array<PackageDependency | null> {
```

Comment:
```suggestion
  sources: ApplicationSource[]
): PackageDependency[] {
```
don't return null inside array